packages/core/src/client.ts

```diff
@@ -331,4 +332,8 @@ export class Client {
     const operation = this.createRequestOperation('mutation', query, opts);
     return this.executeRequestOperation(operation);
   };
+
+  get debugSource() {
```

Comment:
This will need some adjustements to the babel plugin, not entirely sure how `bublé` handles getters might be worth loking into the compiled target here

Reply:
Good point, I've moved the API around so it's back to `client.debugTarget` - does that solve our problem?

Reply:
Yes double checked, it seems to do :)